crates/sui-core/src/transaction_orchestrator.rs

```diff
@@ -392,6 +422,10 @@ where
             good_response,
         )
     }
+
+    pub fn load_all_pending_transactions(&self) -> Vec<VerifiedTransaction> {
```

Comment:
Where is `pending_tx_log` used for recovery, or is it going to be in a subsequent PR?

Reply:
yes